src/v/cluster/self_test/cloudcheck.cc

```diff
@@ -64,6 +65,7 @@ cloudcheck::run(cloudcheck_opts opts) {
           "Cloud storage is not enabled, exiting cloud storage self-test.");
         auto result = self_test_result{
           .name = _opts.name,
+          .test_type = "cloud",
```

Comment:
nit: I don't feel strongly that it needs to be in this PR, but it'd have been nice to see a test that ran the cloud check via RPK and parsed the output to assert that we don't print "IOPS" etc

Reply:
Will add in follow up PR.